charts/dg-docassembly/values.yaml

```diff
@@ -26,7 +26,8 @@ java:
     IDAM_WEBSHOW_WHITELIST: https://em-show-aat.service.core-compute-aat.internal/oauth2/callback
     DM_STORE_APP_URL: http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     DOCMOSIS_TEMPLATES_ENDPOINT: https://docmosis-development.platform.hmcts.net
-    DOCMOSIS_ENDPOINT: https://docmosis-development.platform.hmcts.net/rs/render
+    DOCMOSIS_ENDPOINT: https://docmosis-development.platform.hmcts.net/rs/convert
+    DOCMOSIS_RENDER_ENDPOINT: https://docmosis-development.platform.hmcts.net/rs/render
```

Comment:
why not make the docmosis-endpoint just include the host rather than the operation, so that you can reuse it
https://docmosis-development.platform.hmcts.net/rs/
that way you can just add 'convert' to do conversion and 'render' to do rendering